### DISCUSSION TITLE
✅ 🔗 New menu widget does not have 'none' option for mobile dropdown, unlike old menu widget (Duplicate of #21796)

### DISCUSSION THREAD

**Discussion by joehdevwp** [elementor/elementor#23509] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Really guys? Yet another in a long, long list of *partially* implemented additions and half-solutions in Elementor.  You get 70-80% of the way there, and then stop.

The "old" Wordpress Menu widget has, in the Mobile Dropdown: Breakpoint setting, an option for 'none'. 

The new Menu widget does not. 

Did it not occur to you that it might be useful to add to the new Menu widget a feature that was very useful in the old menu widget?  You've already developed the code. There should be very little to do here. Just do the same thing on the new and make your new widget even more versatile than the old one. But no. 

I don't get it. This is just one example (of many) of common sense things that Elementor doesn't have the sense to do until it's pointed out to them.

### Describe the solution you'd like

Add the option to have no mobile menu dropdown to the new Menu widget. 

Use cases? I don't know, lets see... how about... a Footer Menu? I don't know about you but personally I don't like dropdown menus in my footers. Neither do my clients. Nor does the rest of the world. 

### Describe alternatives you've considered

Ditching Elementor for another page builder.

**Comment by nicholaszein** [comment]
Hi @joehdevwp!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.17.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
